### PR TITLE
auto_dw blog

### PR DESCRIPTION
**Pull request by ryw**
# Notes for blog or docs authors:

-   The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
-   You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
-   Please write a `description: ` in the frontmatter of any new blog or doc
-   Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

### CODE REVIEW

**Comment by ryw** [comment]
@samay-sharma @analyzer1 @DarrenBaldwin07 
please review + feel free to PR improvements

File: src/content/blog/2024-05-13-auto-dw/index.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+slug: auto-dw
+title: 'pg_auto_dw: a Postgres extension to build well-formed data warehouses with AI'
+description: We are working on a new Postgres extension, pg_auto_dw, to build well-formed data warehouses with AI.
+authors: [ryw]
+tags: [postgres, announcement]
```
Comment (DarrenBaldwin07) [inline src/content/blog/2024-05-13-auto-dw/index.mdx:6]: maybe add `extensions`

**Review by DarrenBaldwin07 — APPROVED** [review]
LGTM

**Comment by samay-sharma** [comment]
I think it reads very light to me @ryw . I think adding more context (and examples) could make this pretty popular. We'll need some help from Jonathan as well to define this stuff.

* Announcing that we are working on this - Similar to the intro paragraph we have right now
* Postgres for everything - Postgres has been the de-facto operational store, now people are using it for analytics and DWH (some examples). We also have a stack which enables DWH from Postgres.
* Efficient DWH schemas - One of the key things for DWH performance is the data structure and transformations going from operations to analytics. Give example of Kimball (or something else) as good vs bad schemas.
* Doing transformations in Postgres - If Postgres is your operational store and your analytics store, why not rely on Postgres to do the transformations. With LLMs today, you should be able to do this automatically.
Maybe a paragraph on the how.
* Join us on the journey.

**Comment by ryw** [comment]
@samay-sharma @FloorD I added more content - want to take a pass at improving? Or at least more critique that I can use to improve + clarify?

**Pull request merged**